sys/auto_init/saul/auto_init_apds99xx.c

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
This file is the duplicate of `drivers/saul/init_devs/auto_init_apds99xx.c`. Which one should be kept ?

Reply:
The version in `sys/auto_init` is just a leftover after the rebase. I was sure that I had moved it from `sys/auto_init/saul/` to `drivers/saul/init_devs` instead of copying it. Good catch, thanks.